.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+
+node_js:
+  - "6"
+  - "7"
```

Comment:
7 is not required as it's not relevant anymore with 8 being out and 10 kicking in soon for LTS.